About.html

```diff
@@ -0,0 +1,89 @@
+            <div class="hamburg">
+                <div class="hamburg1"></div>
+                <div class="hamburg2"></div>
+                <div class="hamburg3"></div>
+            </div>
+            <ul class="menu">
+                <li><a class="menu-item" href="index.html">home</a></li>
+                <li><a class="menu-item" href="About.html">About</a></li>
+                <li><a class="menu-item" href="#logo">logo</a></li>
+                <li><a class="menu-item" href="#vision">Vision</a></li>
+            </ul>    
+        </nav>
+    </header>    
+        <section class="grtittle">
+            <h1>Kamulete training center(Rwanda)</h1>
+            <hr class="linebar">
+            <div class="centerpar">
+                <p>kamulete trainig center for all people around the world,<br> we invite you to join our training center.
+                    kamulete training center was created in 2023 by kamulete bertin who is the CEO form Rwanda,<br> he found that to train people by knwowing many languages is very helps full and develops creativness in someone.
+                    with that this trainig center give you a diploma after complited the month acording to your program choosen,<br> that diploma is valid in each country around the word.
+                    we have more than 20 teachers by which we have collaboration and they helps us to give some online classes.
+                </p>
+            </div>
+          <div class="contactclass">
+            <p class="contactclass1">please contact us via email for more information.    </p>
+            <p class="contactclass2">email: Kamuletetrainingcenter988@gmail.com<br></p>
+            <p class="contactclass3">@250 kigali , palace (Rwanda)</p>
+          </div>
+        </section>
+        <section id="logo" class="classlog">
+            <h2>kamulete-training-center-logo</h2>
+            <hr class="linebar">
+            <p>the logo of kamulete traning center logo was decided from a competition from 12 january to 30january.
+                After finishing the competition voting and selection committe's judjjing process,<br> the logo from 20 years oldswon the prize-winning-logo for kamulete training center as follow
+            </p>
+            <img src="img-folder/logo-image.png" alt="">
+        </section>
+        <section id="vision" class="vissonsee">
+            <h2>see our vision and goals </h2>
+            <hr class="linebar">
+            <p>see how we working with our students</p>
+           <div>
+           <p class="section-red">Different community</p>
+            <p class="section-red1">Different community</p>
+            </div>
+        </section>
+        <section class="partners">
+            <h3>partners</h3>
+            <hr class="linebar">
+            <ul>
+                <li>img kivumotor</li>
+                <li>KinMarch</li>
+                <li>Vinkingboy</li>
+                <li>jmaskshop</li>
+            </ul>
+        </section>
```

Comment:
- [ ] Great job adding the partners section as requested by the previous reviewer however please make sure that your partners section alignment matches that of the [design guidelines
](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). 

- [ ] [OPTIONAL]While you are at it, it will be better if you edit the footer design of home page to match the design guideline as well.

Your design:

<img width="328" alt="Screenshot 2023-03-16 at 9 48 34 PM" src="https://user-images.githubusercontent.com/44798044/225727707-1d8c631f-221a-4ef1-8dc9-00b3d81420f3.png">
Behance design:
<img width="343" alt="Screenshot 2023-03-16 at 9 49 13 PM" src="https://user-images.githubusercontent.com/44798044/225727694-f7ed9843-5fc6-421a-88f2-f453da804e1f.png">